server/controllers/devices.coffee

```diff
@@ -61,7 +61,7 @@ deviceExists = (login, cb) ->
         else if body.length is 0
             cb null, false
         else
-            cb null, body[0]
+            cb null, body[0]?.value
```

Comment:
Yes, for example here: https://travis-ci.org/cozy/cozy-proxy/jobs/104924355#L753